fmriprep/workflows/bold/base.py

```diff
@@ -514,6 +514,37 @@ def init_bold_wf(
             (bold_anat_wf, bold_surf_wf, [('outputnode.bold_file', 'inputnode.bold_t1w')]),
         ])  # fmt:skip
 
+        if nonstd_spaces.intersection(('anat', 'T1w')):
+            # Source file should be output T1w-space volumetric file and fsnative2t1w_xfm
+            merge_surface_sources = pe.Node(
+                niu.Merge(2),
+                name='merge_surface_sources',
+                run_without_submitting=True,
+            )
+            workflow.connect([
+                (ds_bold_t1_wf, merge_surface_sources, [('outputnode.bold', 'in1')]),
+                (inputnode, merge_surface_sources, [('fsnative2t1w_xfm', 'in2')]),
+            ])  # fmt:skip
+        else:
+            # sources are bold_file, motion_xfm, boldref2anat_xfm, fsnative2t1w_xfm
```

Comment:
I would include the white-matter and pial surfaces for each hemisphere, since those determine the coordinates at which the BOLD is sampled. If that's difficult to do at the moment, I'm fine with postponing it to another PR.

Reply:
The hemisphere loop happens within `init_bold_surf_wf`, so I need to add those sources there, but I can't pin down where exactly those two files are grabbed in the workflow. I think it'll have to wait for another PR.